snuba/datasets/processors/search_issues_processor.py

```diff
@@ -61,10 +82,10 @@ class SearchIssueEvent(TypedDict, total=False):
     organization_id: int
     project_id: int
     event_id: str
-    group_id: int  # backwards compatibility
-    group_ids: Sequence[int]
+    group_id: int
```

Comment:
Why is this changing to singular `group_id` now? I thought we'd always have multiple ids for non-error issue stuff.

Reply:
Yeah, I had the same discussion with Dan. An occurrence event will be sent for each group(s) that resulted in processing that event. So if multiple Issues are detected for a single issue, then multiple events will be sent to the eventstream.